ChangeLog.txt

```diff
@@ -209,6 +209,30 @@ MAP/HISTORY:
 - Added one historical governor of Mauretaniania Caesariensis
 - Added one historical governor of Lusitania
 - Added three historical governors of Tripolitania
+- Added five historical duces of Thebais
+- Added five historical duces et augustales of Alexandria
+- Added one historical Chalcedonian Patriarch of Alexandria
```

Comment:
These two lines as well.

Reply:
These lines were added for resolving conflicts with ChangeLog.

Reply:
They are no longer relevant, between the time @IhateTrains modified those lines (see below in the file).

Reply:
Yes, I've added many more since then.